packages/dm-core/src/components/Table/Table.tsx

```diff
@@ -118,6 +122,7 @@ export function Table(props: TableProps) {
             <TableHead
               config={config}
               tableVariant={tableVariant}
+              showAdditionalCell={showAdditionalCell}
```

Comment:
perhaps a more descriptive name? 
what kind of cell is this? Functionality cell?

Reply:
I agree 🎉